### PR TITLE
tests: port 2 uc20 part1

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Making some core tests work on uc20

### CODE REVIEW

File: tests/core/enable-disable-units-gpio/task.yaml [hunk 1]
```diff
@@ -10,8 +10,7 @@ details: |
     a reboot the systemd service tries to regenerate the gpio device node if it does not
     find it.
 
-# TODO:UC20: enable for UC20
-systems: [ubuntu-core-1*-64]
+systems: [ubuntu-core-1*-64, ubuntu-core-2*-64]
```
Comment (anonymouse64) [inline tests/core/enable-disable-units-gpio/task.yaml:13]: why not just 

```suggestion
systems: [ubuntu-core-*-64]
```

File: tests/core/create-user/task.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 summary: Ensure that snap create-user works in ubuntu-core
 # FIXME: combine this test with tests/core/create-user-2?
 
-# TODO:UC20: enable for UC20
-systems: [ubuntu-core-1*]
+systems: [ubuntu-core-*]
```
Comment (anonymouse64) [inline tests/core/create-user/task.yaml:4]: this line is now redundant, only ubuntu-core-* systems are available for the core suite

```suggestion
```

File: tests/core/create-user-2/task.yaml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 summary: Ensure create-user functionality
 
-# TODO:UC20: enable on UC20
-systems: [ubuntu-core-1*]
+systems: [ubuntu-core-*]
```
Comment (anonymouse64) [inline tests/core/create-user-2/task.yaml:3]: this line is now redundant, only ubuntu-core-* systems are available for the core suite

```suggestion
```

Comment (sergiocazzolato) [reply tests/core/create-user-2/task.yaml:3]: done

File: tests/core/create-user/task.yaml (hunk 2 above)
Comment (sergiocazzolato) [reply tests/core/create-user/task.yaml:4]: done

File: tests/core/enable-disable-units-gpio/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/core/enable-disable-units-gpio/task.yaml:13]: done

**Comment by sergiocazzolato** [comment]
@anonymouse64 thanks for the review, fixes already pushed.

**Pull request merged**